doc/nrf/dm_ncs_distro.rst

```diff
@@ -0,0 +1,64 @@
+.. _dm_ncs_distro:
+
+Redistributing the |NCS|
+########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This section is intended for users that would like to re-distribute the |NCS| with additions or changes to it.
+It can be useful if you are a software or hardware vendor that would like to augment the |NCS| with additional functionality, and still benefit from the infrastructure it provides.
+
+The contents of this section is not comprehensive.
+You can rather consider it as suggestions for extending the |NCS|.
+You can use other mechanisms as long as they comply with the text in the :file:`LICENSE` file located in the root of every Git repository in the |NCS|.
+If you need any help, reach out to Nordic Semiconductor.
+
+Getting added to the main manifest
+**********************************
+
+The simplest way to become part of the |NCS| is to be included in its official `west manifest file`_.
+This way, your own :ref:`module repository <zephyr:modules>` will automatically be part of the |NCS| official distribution and will be included in all releases.
+
+This solution is reserved for selected vendors and requires a commercial agreement with Nordic Semiconductor.
+Contact your local representative if you wish to be considered for inclusion.
+
+Some examples of this type of integration are:
+
+* :ref:`Memfault <mod_memfault>`.
+  This functionality is present through the addition of a `west project in the nRF Connect SDK manifest <Memfault entry in the manifest_>`_.
+  The `repository provided by Memfault <Memfault firmware SDK_>`_ is public.
+  By default, |NCS| users will obtain a copy of it (as well as the integration code necessary for |NCS| based applications) when :ref:`getting the code <cloning_the_repositories>`.
+
+* ANT protocol support.
+  The code and documentation of this protocol is confidential, and access to the private module repository requires the GitHub user to be added to it.
+  This is why the west project is disabled by default, using the :ref:`project groups <zephyr:west-manifest-groups>` feature of west.
+  This means that the ANT entry in the manifest is disabled by default through its presence in the `manifest group filter`_.
+  You will not get a copy of this repository when :ref:`getting the code <cloning_the_repositories>`, and instead, need to enable the repository first using west itself to fetch it locally::
+
+    west config manifest.group-filter +ant
```

Comment:
2 queries:
1) How does someone enable this from ncs toolchain manager or would it be the same?
2) Is it worth linking to documentation (i.e. git or github) about how a user sets up authentication for the protected repos?

Reply:
> How does someone enable this from ncs toolchain manager or would it be the same?

It would be exactly the same, from the command-line.

> Is it worth linking to documentation (i.e. git or github) about how a user sets up authentication for the protected repos?

This is outside the scope of this section in my opinion, given that this is about redistribution. I think we should add a chapter somewhere in the NCS Getting Started doc mentioning this, will work on it on a separate PR.